packages/ui/src/walkthrough-card/WalkthroughCard.tsx

```diff
@@ -0,0 +1,45 @@
+"use client";
+import { useRouter } from "next/navigation";
+
+import { WalkthroughInfo } from "@repo/data/useWalkthroughData";
+
+import "./WalkthroughCard.css";
+
+export interface WalkthroughCardProps {
+  id: string;
+  data: WalkthroughInfo;
+  href?: string;
```

Comment:
I think this should be required based on our use cases. What would your thoughts be about making this be a property called `walkthroughId` and moving the pathing part with `URL_WALKTHROUGH_HREF` directly into the component. In theory the 2 use cases for this always point to a walkthrough. We could also create a helper function in `url.ts` to replace `URL_WALKTHROUGH_HREF` like we've done with some of the testids like so:

```TypeScript
export const GET_WALKTHROUGH_HREF = (id: string) => `/walkthrough/${id}/`;
```

Reply:
I like the idea of moving it to the card actually. The helper function I don't think is necessary since we already have a constant for the base path for walkthroughs and can just do 
`href={`${URL_WALKTHROUGH_HREF}/${walkthroughId}`}`